initdb.py

```diff
@@ -99,8 +92,8 @@ def connect_to_database(connection):
     """
     try:
         connection.database = DB_NAME
-    except mysql.connector.Error as mysqlError:
-        raise Exception('There was a problem connecting to the database {}:\n{}'.format(DB_NAME, mysqlError))
+    except mysql.connector.Error as MySqlError:
```

Comment:
`MySqlError` violates [Google Python Style Guide: Naming](https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming)

make it something like: `mysql_error`